guess.js

```diff
@@ -9,78 +9,10 @@ function ask(questionText) {
 
 start();
 
-let max= process.argv[2] ||100;
-let min=0;
-
-async function start(){
-
-  console.log(`\nPick a number between 1 and ${process.argv[2] ||100} and I'll try to guess it.\n`)
+async function start() {
+  console.log("Let's play a game where you (human) make up a number and I (computer) try to guess it.")
   let secretNumber = await ask("What is your secret number?\nI won't peek, I promise...\n");
-  console.log('You entered: ' + secretNumber + '\n');
-  
-  if(secretNumber>process.argv[2]){
-    console.log('************INVALID NUMBER******************');
-    start();;
-  }
-  guess1=(Math.random()*(max-min)+min);
-
-  isItNum(min, max, guess1);
-}
-  
-async function isItNum(min,max,guess){   
-  let ans1 = await ask(`Is it...${Math.ceil(guess)}? (Y) or (N)`);
-
-    if(ans1.toUpperCase()==='N'){
-    isHighLow(min, max, guess)  
-
-    } else if (ans1.toUpperCase()==='Y'){ console.log(`\n********Your number is ${Math.ceil(guess)}!**********\n`);
-      compguess();
-
-    } else console.log('***INVALID RESPONSE, (Y) or (N)***')//if (ans1.toUpperCase()!=='Y'||ans1.toUpperCase()!=='N') {console.log('INVALID RESPONSE, (Y) or (N)')
-    isItNum(min,max,guess);
-}
-  
-async function isHighLow(min,max,guess){
-   let ans= await ask(`Is it higher (H), or lower (L)?`); 
-
-      if(ans.toUpperCase()=== 'H' && max > min){
-        min=Math.floor(guess);
-      }else if(ans.toUpperCase()==='L'&& max>min){
-        max=Math.ceil(guess);
-      }else if(max < min){ 
-        console.log((`HOLD ON, YOU SAID ${guess} 
-        WAS LESS THAN ${max} AND GREATER THAN ${min}`)); isHighLow(min,max,guess) 
-      }else if (ans!=='H'||ans!=='L' || ans==undefined) isHighLow(min,max,guess);
```

Comment:
another way to check if `ans` is a falsy value would be to say `!ans` rather than `ans === undefined` that'll also handle if the answer is `NaN`, `''`, `null`, or `false`